pkg/result/filter.go

```diff
@@ -210,14 +211,39 @@ func getIgnoredIDs(ignoreFile string) []string {
 		if strings.HasPrefix(line, "#") || line == "" {
 			continue
 		}
-		ignoredIDs = append(ignoredIDs, line)
+		// Process all fields
+		fields := strings.Fields(line)
+		if len(fields) > 1 {
+			exp, err := getExpirationDate(fields)
+			if err != nil {
+				log.Logger.Warnf("Error while parsing expiration date in .trivyignore file: %s\n", err)
```

Comment:
nit: you don't need `\n`.

```suggestion
				log.Logger.Warnf("Error while parsing expiration date in .trivyignore file: %s", err)
```